src/routes/AuthenticatedRoute.jsx

```diff
@@ -0,0 +1,19 @@
+import { UserDetail, UserList} from '@/pages/users'
+import { Dashboard } from '@/pages/authenticated'
+
+const AuthenticatedRoute = [
```

Comment:
rename to ProtectedRoutes as well as change name in index.js file (where it is imported)